df.ui-menus.xml

```diff
@@ -2525,51 +2525,37 @@
 
         <compound name='mod_manager'>
             <stl-vector pointer-type='mod_headerst' name='mod_header'/>
+            -- Begin Steam-only stuff
+            -- These fields exist in other versions but aren't actually used
+            <pointer name='subscribed_file_id'/>
```

Comment:
Idea: we have the option of putting these behind a union with `has-bad-pointers='true'` to discourage tools from traversing potentially invalid pointers. I'm not sure if the pointers here are zero-initialized.

Reply:
It might still be a good idea to use `has-bad-pointers='true'`, but I was able to confirm that they are intitialized to NULL in DF Classic 50.08
![image](https://github.com/DFHack/df-structures/assets/977482/5244267d-8584-467b-9989-ea9ce83d0203)

Reply:
Well, that doesn't mean they're _always_ initialized to null - for that, we would have to double-check the disassembly. But I believe it is a safe assumption for globals like this.